README.md

```diff
@@ -14,17 +14,48 @@ use following snippets:
 
 ## Dasharo CPU HCL generation
 
+1. Download the `dasharo_hcl_reports` folder available on [Cloud](https://cloud.3mdeb.com/index.php/f/467774) 
+and unzip it.
```

Comment:
This is an internal link, so we should avoid putting it in a public repository. Please remove.

Reply:
@pietrushnic fixed ac9347f3b1f463030efa589417b5bf330f4a6871